include/tdi/arch/tna/tna_defs.h

```diff
@@ -37,7 +37,7 @@ typedef uint32_t tna_pipe_id_t;
 #define TNA_INVALID_PIPE 0xFFFFFFFF
 
 /** Identifies a pipeline on an ASIC.  Note dev_pipe_id can be set to
- *  TNA_DEV_PIPE_ALL as a special value to indicate "all pipelines". */
+ *  BF_DEV_PIPE_ALL as a special value to indicate "all pipelines". */
```

Comment:
Why change the comment but not the define below? Is this an error?

Reply:
Yeah they were changed by mistake. Reverted back